modepy/matrices.py

```diff
@@ -78,8 +76,11 @@
 """
 
 
+BasisFunctionType = Callable[[np.ndarray], np.ndarray]
```

Comment:
```suggestion
NodalFunction: TypeAlias = Callable[[np.ndarray], np.ndarray]
```
?
So two things:

- make it a type alias
- rename